CE04OSSM/CE04OSSM_D00017_ingest.csv

```diff
@@ -0,0 +1,24 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+# mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/cpm2/syslog/cpm_status.txt,CE04OSSM-RIC21-00-CPMENG000,telemetered,Pending,Incorrectly specified driver and dataset
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl26/syslog/*.syslog.log,CE04OSSM-RID26-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+mi.dataset.driver.adcpt_acfgm.dcl.pd0.adcpt_acfgm_dcl_pd0_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl26/adcpt/*.adcpt.log,CE04OSSM-RID26-01-ADCPTC000,telemetered,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl26/velpt2/*.velpt*.log,CE04OSSM-RID26-04-VELPTA000,telemetered,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl26/phsen/*.phsen.log,CE04OSSM-RID26-06-PHSEND000,telemetered,Available,
+mi.dataset.driver.suna.suna_dcl_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl26/nutnr/*.nutnr.log,CE04OSSM-RID26-07-NUTNRB000,telemetered,Available,
+mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl26/spkir/*.spkir.log,CE04OSSM-RID26-08-SPKIRB000,telemetered,Available,
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl27/syslog/*.syslog.log,CE04OSSM-RID27-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl27/optaa*/*.optaa.log,CE04OSSM-RID27-01-OPTAAD000,telemetered,Available,
```

Comment:
In the CE04OSSM/X00017/dcl27/ the optaa directory is labeled "optaa_decimated", will it be labeled the same in the D00017/dcl27/ or will it be named "optaa" as normal?

Reply:
No, the iridium modems all are set to transfer the optaa data in a "decimated" mode. The file glob above works with folders that have `_decimated` added.